static/bidder-params/zeta_global_ssp.json

```diff
@@ -4,5 +4,10 @@
   "description": "A schema which validates params accepted by the Zeta Global SSP adapter",
 
   "type": "object",
-  "properties": {}
+  "properties": {
+    "sid": {
+      "type": "integer",
+      "description": "An ID which identifies the publisher"
+    }
```

Comment:
Is this parameter required? Can it be empty?
If it is required, please add `"required": ["sid"]`. 
[Example](https://github.com/prebid/prebid-server/blob/master/static/bidder-params/33across.json)